test-data/unit/semanal-basic.test

```diff
@@ -8,8 +8,9 @@ x
 [out]
 MypyFile:1(
   AssignmentStmt:1(
-    NameExpr(x* [__main__.x])
-    IntExpr(1))
+    NameExpr(x [__main__.x])
```

Comment:
Changes such as these are a side effect of making semantic analyzer tests use the constant folding logic. Previously it was disabled due to historical reasons. The actual behavior outside tests wasn't changed (for most of these test changes).